From 49757cf8e58ca833d07ede0ec640bf288a667bd2 Mon Sep 17 00:00:00 2001
From: Thu Ba Nguyen <tbnguyen@amperecomputing.com>
Date: Thu, 15 Apr 2021 11:44:28 +0700
Subject: [PATCH] EntityManager: Add new platform Mt.Jade

Added a new Ampere Mt.Jade platform configuration file.
It includes adc, temp, fan sensors.

Signed-off-by: Thu Ba Nguyen <tbnguyen@amperecomputing.com>
---
 configurations/Mt_Jade.json | 2 ++
 meson.build                 | 1 +
 2 files changed, 3 insertions(+)
 create mode 100644 configurations/Mt_Jade.json

diff --git a/configurations/Mt_Jade.json b/configurations/Mt_Jade.json
new file mode 100644
index 0000000..2c63c08
--- /dev/null
+++ b/configurations/Mt_Jade.json
@@ -0,0 +1,2 @@
+{
+}
diff --git a/meson.build b/meson.build
index ffe3d09..67dc945 100644
--- a/meson.build
+++ b/meson.build
@@ -103,6 +103,7 @@ configs = [
     'Intel Front Panel.json',
     'Nisqually.json',
     'NVME P4000.json',
+    'Mt_Jade.json',
     'PCIE SSD Retimer.json',
     'PSSF132202A.json',
     'PSSF162205A.json',
-- 
2.17.1

